CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Improved thread display
 - Expose ban information for current user (`User#getBanned`)
 - Bugfix on attachment size
+- Add `editMessage` functions to `ChannelViewModel` (Adrian)
```

Comment:
can you yank this out? makes it harder to merge

Reply:
Resolved conflicts the line.